sdk/identity/azure-identity/azure/identity/_credentials/azure_arc.py

```diff
@@ -87,12 +87,12 @@ def _get_policies(config, **kwargs):
 
 def _get_request(url, scope, identity_config):
     # type: (str, str, dict) -> HttpRequest
-    if identity_config != {}:
+    if identity_config:
         raise ClientAuthenticationError(
-            message="User assigned identity is not supported by the Azure Arc Managed Identity Endpoint. To " \
-                    "authenticate with the system assigned identity omit the client id when constructing the" \
-                    " ManagedIdentityCredential, or if authenticating with the DefaultAzureCredential ensure" \
-                    " the AZURE_CLIENT_ID environment variable is not set."
+            message="User assigned managed identities are not supported by Azure Arc. To authenticate with the " \
```

Comment:
Could you please format with `black`? (It will at least remove the backslashes)

Reply:
When I used `black` it put the full string on one line, which pylint was unhappy about -- should I do that and suppress a warning?